sql/src/test/java/io/crate/operation/collect/files/CSVLineParserTest.java

```diff
@@ -22,95 +24,123 @@ public void setup(){
 
     @Test(expected = IllegalArgumentException.class)
     public void parse_givenEmptyHeader_thenThrowsException() throws IOException {
-        subjectUnderTest.parseHeader("\n");
+        String header = "\n";
+        BufferedReader bufferedReader = new BufferedReader(new StringReader(header));
+        subjectUnderTest.parseHeader(bufferedReader);
         subjectUnderTest.parse("GER,Germany\n");
     }
 
     @Test(expected = IllegalArgumentException.class)
```

Comment:
fine with me but for the future, we usually use JUnit's `expectedException` from the `CrateUnitTest` base class.